nautobot/core/tests/test_views.py

```diff
@@ -572,3 +574,24 @@ def test_access_for_superuser(self):
 
         # Check for success status code (e.g., 200)
         self.assertEqual(response.status_code, 200)
+
+
+class TimeZoneTestCase(TestCase):
+    def test_timezone_change(self):
+        self.user.is_superuser = True
+        self.user.save()
+        self.client.force_login(self.user)
+
+        timezone_name = timezone.get_current_timezone_name()
+        new_timezone_name = "US/Eastern"
+        form_data = {"request_profiling": False, "timezone": new_timezone_name}
+        url = reverse("user:advanced_settings_edit")
```

Comment:
Feels weird for this to be under "Advanced Settings", especially since once set it appears in the table under "Preferences" - maybe need to rethink the UI/UX of this?

Reply:
Any suggestion?

Reply:
I wonder if an entirely separate tab makes more sense here. Since it would be hard to set the setting from the "preferences: tab

Reply:
Per standup, lets add a separate form for timezone under the "Preferences" tab.